openquake/calculators/tests/outputs/ruptures.csv

```diff
@@ -1,3 +1,3 @@
-#										"generated_by='OpenQuake engine 3.7.0-gitab830d1d1c', start_date='2019-09-14T06:52:28', checksum=87673300, investigation_time=200.0, ses_per_logic_tree_path=3"
-rupid	multiplicity	mag	centroid_lon	centroid_lat	centroid_depth	trt	strike	dip	rake	boundary
-1067	1	5.850000E+00	-1.222000E+02	3.690000E+01	6.000000E+00	Active Shallow Crust	4.497391E+01	9.000000E+01	0.000000E+00	MULTIPOLYGON(((-122.24072 36.86742 2.585,-122.15924 36.93257 2.585,-122.15924 36.93257 9.415,-122.24072 36.86742 9.415,-122.24072 36.86742 2.585)))
+#,,,,,,,,,"generated_by='OpenQuake engine 3.8.0-git7effe21d62', start_date='2019-11-20T16:24:51', checksum=369626030, investigation_time=1.0, ses_per_logic_tree_path=600"
+rupid,multiplicity,mag,centroid_lon,centroid_lat,centroid_depth,trt,strike,dip,rake
```

Comment:
`rupid` → `rup_id` to match the events table

Reply:
Let's do that in another PR.